### PR TITLE
Update managing-channels.rst

### PR DESCRIPTION
**Pull request by Sayanta66**
Adding an extra point in the Leaving a Channel section which is available in Mattermost Desktop

### CODE REVIEW

**Comment by Sayanta66** [comment]
I hope this addition will really help as I saw it was missing, Plus I am taking on the #4487 issue. I will work on the documentation.

File: source/messaging/managing-channels.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ For example, for the following channel, ``https://community.mattermost.com/core/
 Leaving a channel
 -----------------
 
-Select the channel name at the top of the center pane to access the drop-down menu, then select **Leave Channel**. Any team member who leaves a Private channel must be re-added by another channel member if they wish to rejoin. Team members will not receive mention notifications from channels of which they are not members.
+Select the channel name at the top of the center pane to access the drop-down menu, then select **Leave Channel**. For Mattermost Desktop, you can also do that without entering the channel. If you wish to leave a channel, simply move your cursor on the top of the channel name at the sidebar. There, you can see the three dots icon (channel options) beside the channel name. Click the icon to access the drop down menu, then select **Leave Channel**. Any team member who leaves a Private channel must be re-added by another channel member if they wish to rejoin. Team members will not receive mention notifications from channels of which they are not members.
```
Comment (justinegeffen) [inline source/messaging/managing-channels.rst:72]:
```suggestion
From within the channel, select the channel name at the top of the center pane to access the drop-down menu, then select **Leave Channel**. For Mattermost Desktop, you can hover over the channel name in the sidebar. Select **Channel Options** (the three dots), then select **Leave channel**. 

Any team member who leaves a Private channel must be re-added by another channel member if they wish to rejoin. Team members will not receive mention notifications from channels of which they are not members. 
```

File: source/messaging/managing-channels.rst [hunk 2]
```diff
@@ -154,4 +154,4 @@ Navigating recently viewed channels
 Use the **History** arrows at the top of the sidebar to move back and forth through your channel history. 
 
 - Select the left arrow to go back one page. 
-- Select the right arrow to go forward one page.
+- Select the right arrow to go forward one page.
```
Comment (justinegeffen) [inline source/messaging/managing-channels.rst:157]:
```suggestion
- Select the right arrow to go forward one page.
```

Comment (justinegeffen) [reply source/messaging/managing-channels.rst:157]: We don't use trailing white spaces in our documentation - but I am happy to discuss the reason behind this change if you're open to it?

File: source/messaging/managing-channels.rst (hunk 1 above)
Comment (justinegeffen) [reply source/messaging/managing-channels.rst:72]: This is a great addition. 

I've edited it for brevity, as we work on the principle of "less is more" in our docs. Hope that makes sense!

File: source/messaging/managing-channels.rst (hunk 2 above)
Comment (Sayanta66) [reply source/messaging/managing-channels.rst:159]: It was an error from my side. Sorry for the whitespace.

**Comment by cwarnermm** [comment]
Thanks for this PR, @Sayanta66! We really appreciate the effort you've put in. Currently the documentation adequately covers this use case, so this update is not going to be applied at this time.

**Pull request closed without merging**